core/src/main/java/com/linecorp/armeria/common/NonWrappingRequestContext.java

```diff
@@ -67,18 +69,20 @@
      * Creates a new instance.
      *
      * @param sessionProtocol the {@link SessionProtocol} of the invocation
+     * @param uuid the {@link UUID} associated with this context
      * @param req the {@link HttpRequest} associated with this context
      * @param rpcReq the {@link RpcRequest} associated with this context
      */
     protected NonWrappingRequestContext(
             MeterRegistry meterRegistry, SessionProtocol sessionProtocol,
-            HttpMethod method, String path, @Nullable String query,
+            HttpMethod method, String path, UUID uuid, @Nullable String query,
             @Nullable HttpRequest req, @Nullable RpcRequest rpcReq) {
 
         this.meterRegistry = requireNonNull(meterRegistry, "meterRegistry");
         this.sessionProtocol = requireNonNull(sessionProtocol, "sessionProtocol");
         this.method = requireNonNull(method, "method");
         this.path = requireNonNull(path, "path");
+        this.uuid = requireNonNull(uuid, "uuid");
```

Comment:
Ditto - Please move this assignment right before `method` assignment. Same for all similar changes in this PR.